### DISCUSSION TITLE
26.4% for perfect match on Missing Slice

### DISCUSSION THREAD

**Discussion by VAggrippino** [cssbattle/forum#64] [Bugs reports]
This looks like a perfect match, but CSSBattle only gave me 26.4%:
Battle: [Missing Slice](https://cssbattle.dev/play/6)

It appears that this is target number 6 of battle number 1, but I'm not certain I understand the hierarchy of these challenges correctly.

![20210612_115840_CSSBattle-26 4-match](https://user-images.githubusercontent.com/195491/121764485-ed22ea80-cb76-11eb-83f8-1f6a389a3701.png)

Could it be that whatever is used to take the _screenshot_ doesn't work with `clip-path`?

```html
<style>
  * {
    margin: 0;
    position: absolute;
  }
  html {
    background: #E3516E;
    clip-path: circle(100px);
    height: 100vh;
    width: 100vw;
  }
  body {
    left: 100px;
    top: 50px;
    height: 100px;
    width: 100px;
    background: #51B5A9;
  }
  ::before, ::after {
    content: '';
    height: 100%;
    width: 100%;
    position: absolute;
  }
  body::before {
    background: #FADE8B;
    left: 100%;
  }
  body::after {
    background: #F7F3D7;
    top: 100%;
  }
</style>
```

I did this on Firefox, but I tested and got the same results in Chrome

**Comment by chinchang** [comment]
Seems like a problem in latest Chromium with `clip-path`. This is what the `clip-path` is doing to your `html`: 
<img width="568" alt="Screenshot 2021-06-12 at 4 12 14 PM" src="https://user-images.githubusercontent.com/379918/121773360-16ed0900-cb99-11eb-8fdd-50e94ac46181.png">

Let us see if we can revert to a previous version till its fixed

**Reply by chinchang** [reply]
Also, this shows up fine in the preview, because the behaviour is different for normal window and iframe.
